autode/opt/optimisers/rfo.py

```diff
@@ -83,25 +83,42 @@ def _low_level_cart_hessian(self) -> np.ndarray:
         see e.g. (https://manual.q-chem.com/5.2/A1.S2.html). To ensure this
         condition is satisfied
         """
+        logger.info("Calculating low-level Hessian")
 
         species = self._species.copy()
         species.calc_hessian(method=get_lmethod(),
                              n_cores=self._n_cores)
 
         return species.hessian
 
-    def _sanitised_step(self, delta_s: np.ndarray) -> np.ndarray:
-        """Ensure the step to be taken isn't too large"""
+    def _take_sanitised_step(self,
```

Comment:
Sanitised seems like an odd world to use here

Reply:
fair